migrator/migrations/m_206_to_m_207_add_default_policy_edge/migration.go

```diff
@@ -0,0 +1,84 @@
+// Code generated by make bootstrap_migration generator. DO NOT EDIT.
+package m206tom207
+
+import (
+	"context"
+
+	"github.com/stackrox/rox/pkg/uuid"
+	"github.com/pkg/errors"
+	"github.com/stackrox/rox/migrator/migrations/m_206_to_m_207_add_default_policy_edge/schema"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/migrator/migrations"
+	"github.com/stackrox/rox/migrator/types"
+	"github.com/stackrox/rox/pkg/sac"
+	"gorm.io/gorm"
+)
+
+const (
+	startSeqNum = 206
+)
+
+var (
+	migration = types.Migration{
+		StartingSeqNum: startSeqNum,
+		VersionAfter:   &storage.Version{SeqNum: int32(startSeqNum + 1)},
+		Run: func(databases *types.Databases) error {
+			err := updatePolicyCategoryEdge(databases.GormDB)
+			if err != nil {
+				return errors.Wrap(err, "inserting policy categpry edge")
+			}
+			return nil
+		},
+	}
+)
+
+func updatePolicyCategoryEdge(db *gorm.DB) error {
```

Comment:
What happens if the customer added the category back to the default policy manually before the migration runs?